app/controllers/samples_controller.rb

```diff
@@ -667,8 +677,7 @@ def show
     @host_genome = @sample.host_genome ? @sample.host_genome : nil
     @background_models = current_power.backgrounds.where(ready: 1)
     @can_edit = current_power.updatable_sample?(@sample)
-    @git_version = ENV['GIT_VERSION'] || ""
-    @git_version = Time.current.to_i if @git_version.blank?
+    @git_version = ENV['GIT_VERSION'] || "" # used for cache busting
```

Comment:
Does this mean the cache will be invalidated every time we deploy a new commit? I think we would need more persistence than that?

Reply:
Yes it does. It is conservative in that way. I'd prefer that for now. Keep in mind also that the cache will be regenerated on any report load, not just precaching.

Reply:
an alternative way to do this is with a constant in initializers called WEB_REPORT_CACHE_VERSION, every time there's a change in format, up the version.

Reply:
Yea, that's what Charles suggested as well. If people here feel it's worth the risk then I will implement manually versioning of the cache.